src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int NAME = 0;
+    private static final int AMOUNT = 1;
+    private static final String EXCEPTION_READ_TEXT = "Can't read the file";
+    private static final String EXCEPTION_WRITE_TEXT = "Can't writ to the file";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String textToWrite = readFile(fromFileName);
+        writeFile(textToWrite, toFileName);
+    }
+
+    private String readFile(String readFileName) {
+        File fileToRead = new File(readFileName);
+        int supply = 0;
+        int buy = 0;
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileToRead))) {
+            String value = reader.readLine();
+            while (value != null) {
+                String[] splitValue = value.split(",");
+                if (splitValue[NAME].equals("supply")) {
+                    supply += Integer.parseInt(splitValue[AMOUNT]);
+                } else {
+                    buy += Integer.parseInt(splitValue[AMOUNT]);
```

Comment:
The current logic assumes that any entry not labeled 'supply' is 'buy'. It might be safer to explicitly check for 'buy' and handle unexpected values appropriately, perhaps by logging a warning or throwing an exception.